fonts-languages/fonts-in-mpdf-7-x.md

```diff
@@ -23,12 +23,12 @@ and `Mpdf\Language\ScriptToLanguage` implementation
 You have 2 font files `Frutiger-Normal.ttf` and `FrutigerObl-Normal.ttf` which you want to be available in mPDF,
 and you will refer to them in HTML/CSS as `frutiger`.
 
-1. Define the directory with the font via `fontDir` configuration key or add it after instatiantion
-   with `$mpdf->AddFontDirectory()` method
+1.  Define the directory with the font via `fontDir` configuration key or add it after instantiation
+    with `$mpdf->AddFontDirectory()` method
```

Comment:
Why is this exactly?

Reply:
It looks like a minor typo, i.e. http://www.dictionary.com/browse/instantiation

Reply:
And the additional space is to improve the layouting of code blocks. It appears that without the space the code in the code blocks is prefixed with a space.

Reply:
I get that, that's ok and cool, but why the doubled space? It occurs on multiple places in the PR.

Reply:
I do not yet understand exactly what you mean by 'doubled space'?
You mean the spaces before the lines below `1.  `? these spaces are just to align the lines below the numbers with the numbers. Is supported by markdown, and looks more clear in the editor...